### PR TITLE
Add remediation script for Azure_AppService_DP_Dont_Allow_HTTP_Access Control

### PR DESCRIPTION
**Pull request by gvaradarajan-msft**
Add remediation script for Azure_AppService_DP_Dont_Allow_HTTP_Access Control.
Make formatting changes to the remediation script for Azure_Subscription_AuthZ_Remove_Management_Certs Control.

### CODE REVIEW

File: Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1 [hunk 1]
```diff
@@ -0,0 +1,917 @@
...
+
+        .EXAMPLE
+        PS> Setup-Prerequisites
+
+        .LINK
+        None
+    #>
+
+    # List of required modules
+    $requiredModules = @("Az.Accounts", "Azure")
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1:92]: We might need to add Az.WebApp module.

File: Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1 [hunk 2]
```diff
@@ -0,0 +1,917 @@
...
+            # Get all non-production slots for this App Service.
+            $nonProductionSlotConfigurations = Get-AzWebAppSlot -ResourceGroupName $resourceGroupName -Name $resourceName
+
+            $isHttpsEnabledOnAllNonProductionSlots = -not $($nonProductionSlotConfigurations.HttpsOnly -contains $false)
+
+            if ($isHttpsEnabledOnProductionSlot -and $isHttpsEnabledOnAllNonProductionSlots)
+            {
+                $appServicesWithHttpsEnabled += $appServiceResource
+                Write-Host "HTTPS is enabled on the production slot and all non-production slots in the App Service." -ForegroundColor $([Constants]::MessageType.Update)
+                return
```
Comment (TarunKrShukla) [inline Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1:315]: Plz check will return just skip the current iteration of loop or returns from loop entirely.

File: Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1 (hunk 1 above)
Comment (gvaradarajan-msft) [reply Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1]: Good catch. Added `Az.Websites` to the list of required modules.

File: Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1 (hunk 2 above)
Comment (gvaradarajan-msft) [reply Scripts/RemediationScripts/Remediate-EnableHTTPSForAppServices.ps1:315]: `return` will skip the current iteration of a `ForEach-Object` block. Reference: https://stackoverflow.com/questions/7760013/why-does-continue-behave-like-break-in-a-foreach-object

**Review by TarunKrShukla — APPROVED** [review]

**Pull request merged**